tendrl/ceph_integration/central_store/__init__.py

```diff
@@ -44,3 +44,6 @@ def save_ecprofile(self, ec_profile):
 
     def save_utilization(self, utilization):
         tendrl_ns.etcd_orm.save(utilization)
+
+    def save_job(self, job):
+        tendrl_ns.etcd_orm.save(job)
```

Comment:
@r0h4n without this function ceph integration can't create or update jobs, job class is in common but it is stored  using save_job function in all components.